mindmaps-engine/src/main/java/io/mindmaps/engine/controller/VisualiserController.java

```diff
@@ -105,11 +109,11 @@ private String matchQuery(Request req, Response res) {
             final JSONArray halArray = new JSONArray();
 
             withGraph(graph).parseMatch(req.queryParams(REST.Request.QUERY_FIELD))
-                    .stream()
+                    .parallelStream()
```

Comment:
I don't get any exception using parallelStream(). How can I check the number of TitanTransactions open? I am using core in parallel basically, by doing:
`entity.relations()` and similar operations